src/web/app/components/teammates-common/notification-style-class.pipe.ts

```diff
@@ -0,0 +1,38 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { NotificationStyle } from '../../../types/api-output';
+
+/**
+ * Two available colors for notification font, depending on the style
+ */
+enum NotificationFontColor {
+  WHITE = 'text-white',
+  DARK = 'text-dark',
+}
+
+/**
+ * Maps between notification style and font color.
+ */
+const fontColorMapping: Record<NotificationStyle, string> = {
+  PRIMARY: NotificationFontColor.WHITE,
+    SECONDARY: NotificationFontColor.WHITE,
+    SUCCESS: NotificationFontColor.WHITE,
```

Comment:
Indentation is abit off, I think it's two spaces only if I'm not wrong.

Reply:
Thank you for pointing it out. I have edited it.